docs/source/conf.py

```diff
@@ -159,9 +159,9 @@
     r"https://doi.org/10.1088/1367-2630/12/7/075008",
     r"https://doi.org/10.7566/JPSJ.90.032001",
     r"https://doi.org/10.7566/jpsj.90.032001",
-    r"https://dl.acm.org/doi/10.1145/3466752.3480059",
     r"https://doi.org/10.1145/3466752.3480059",
-    r"https://doi.org/10.7566/jpsj.90.032001",
+    r"https://dl.acm.org/doi/abs/10.1145/3503222.3507739",
+    r"https://www.sciencedirect.com/science/article/abs/pii/S037704271930344"
```

Comment:
Are these changes because there were some more links broken?

Reply:
At one point yes, but I didn't mean for any changes beyond what Purva fixed in her PR. Changing it back to match `master` now.